bindings/pydeck/examples/01 - Introduction.ipynb

```diff
@@ -21,9 +21,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 11,
```

Comment:
I don't yet grok dealing with Jupyter Notebooks in version control. When I commit these, I try to clear out the output cells before I do. 

We agree that it feels like the task for a linter.